instaclone/src/components/PostContainer.js

```diff
@@ -18,20 +17,17 @@ const PostContainer = (props)=>{
                                 <img className="thumb"  src={data.thumbnailUrl} alt="tumbnail"/>
                             </div>
                         </div>
-                        <img className="imageurl"src={data.imageUrl} alt="rdm" />
-                        <h6>{data.likes} Likes</h6>
-                        <Comment comments={data.comments} />
+                        <img className="imageurl"src={data.imageUrl} alt={data.username} />
+                        <h6>{props.countLikes} </h6>
```

Comment:
I highly recommend naming your prop with the same name as what you're passing down from state. This is far more semantic.

Like so:
```suggestion
                        <h6>{props.likes} </h6>
```